src/Splits.sol

```diff
@@ -160,29 +152,31 @@ abstract contract Splits {
         _assertCurrSplits(accountId, currReceivers);
         SplitsBalance storage balance = _splitsStorage().splitsStates[accountId].balances[erc20];
 
-        collectableAmt = balance.splittable;
-        if (collectableAmt == 0) {
+        // Clear the bits outside of the mask
+        uint128 splittable = balance.splittable & _SPLITTABLE_MASK;
+        if (splittable == 0) {
             return (0, 0);
         }
-        balance.splittable = 0;
+        // Set the value of `splittable` to `0`,
+        // and the bits outside of the mask to `1` to keep the storage slot non-zero.
+        balance.splittable = ~_SPLITTABLE_MASK;
```

Comment:
Wouldn't it be cheaper to just make

```solidity
balance.splittable = 2**127;
```

Reply:
It's treated as a constant anyway, so it's all the same. I think that using `_SPLITTABLE_MASK` here is a bit cleaner than the magic number `2**127`.